src/lib/userStorage/UserProfileStorage.js

```diff
@@ -435,13 +461,23 @@ export class UserProfileStorage implements ProfileStorage {
     return getUserModel(displayProfile)
   }
 
+  /**
+   * Returns field privacy
+   * @param field
+   * @returns {"private"|"public"|"masked"|string}
```

Comment:
i believe we have a type for that

Reply:
changed in d65c060a3868c252b0641e1aa2f9c28d663e1efa